frontend/app/dashboard/src/views/webshops/WebshopsMenu.vue

```diff
@@ -0,0 +1,263 @@
+    <STNavigationBar title="Verkoop"></STNavigationBar>
+    <main>
+      <h1>Webshops</h1>
+
+      <STList class="">
+        <!-- webshops -->
+        <template v-if="enableWebshopModule && visibleWebshops.length > 0">
+          <STListItem
+            element-name="button"
+            :selectable="true"
+            v-for="webshop in visibleWebshops"
+            :key="webshop.id"
+            :class="{
+              selected: isSelected('webshop-' + webshop.id),
+            }"
+            @click="openWebshop(webshop)"
+          >
+            <h2 class="style-title-list">{{ webshop.meta.name }}</h2>
+            <template #right>
+              <span
+                v-if="isWebshopOpen(webshop)"
+                class="icon dot green right-icon small"
+              ></span
+            ></template>
+          </STListItem>
+        </template>
+      </STList>
+
+      <!-- other -->
+      <template
+        v-if="
+          enableWebshopModule &&
+          ((fullAccess && hasWebshopArchive) || canCreateWebshops)
+        "
+      >
+        <hr />
+        <STList>
+          <!-- archive -->
+          <STListItem
+            v-if="fullAccess && hasWebshopArchive"
+            element-name="button"
+            :selectable="true"
+            :class="{ selected: isSelected(Button.Archive) }"
+            @click="$navigate(Routes.Archive)"
+          >
+            <template #left><span class="icon archive"></span></template>
+            <h2 class="style-title-list">Archief</h2>
+          </STListItem>
+
+          <!-- add webshop -->
+          <STListItem
+            v-if="canCreateWebshops"
+            element-name="button"
+            :selectable="true"
+            @click="addWebshop()"
+          >
+            <template #left><span class="icon add"></span></template>
+            <h2 class="style-title-list">Webshop</h2>
+          </STListItem>
+        </STList>
+      </template>
+    </main>
+  </div>
+</template>
+
+<script setup lang="ts">
+import {
+  defineRoutes,
+  useNavigate,
+  useUrl,
+} from "@simonbackx/vue-app-navigation";
+import { useContext, useOrganization } from "@stamhoofd/components";
+import {
+  AccessRight,
+  WebshopPreview,
+  WebshopStatus,
+} from "@stamhoofd/structures";
+import { Formatter, Sorter } from "@stamhoofd/utility";
+import { ComponentOptions, computed, onActivated, ref } from "vue";
+
+//#region composables
+const $organization = useOrganization();
+const $context = useContext();
+const $navigate = useNavigate();
+const urlHelpers = useUrl();
+//#endregion
+
+//#region refs
+const selectedWebshop = ref<string | null>(null);
+const currentlySelected = ref<string | null>(null);
+//#endregion
+
+//#region enums
+enum Button {
+  AddWebshop = "add-webshop",
+  Archive = "webshop-archive",
+  Signup = "signup",
+}
+//#endregion
+
+//#region computed
+const enableWebshopModule = computed(
+  () => $organization.value?.meta.modules.useWebshops ?? false
```

Comment:
```suggestion
  () => $organization.value?.meta.packages.useWebshops ?? false
```
Has been renamed since the DashboardMenu was written